static/templates/single_message.handlebars

```diff
@@ -33,7 +46,7 @@
         <div class="message_content">{{#unless status_message}}{{#if use_match_properties}}{{{msg/match_content}}}{{else}}{{{msg/content}}}{{/if}}{{/unless}}</div>
         {{#if last_edit_timestr}}
           {{#unless include_sender}}
-          <div class="message_edit_notice" title="Edited ({{last_edit_timestr}})">{{t "EDITED" }}</div>
+              <div class="message_edit_notice" title="Edited ({{last_edit_timestr}})">{{t "EDITED" }}</div>
```

Comment:
I think it's ok. its like in python:
```
if:
    something
```
not:
```
if:
something
```

I've made some code formatting fixes as well.